tensorflow/python/ops/embedding_ops.py

```diff
@@ -14,6 +14,7 @@
 # ==============================================================================
 """Operations for embeddings."""
 
+from tensorflow.python.compat import compat
```

Comment:
We need to add this dependency to the BUILD file for embedding_ops

Reply:
Done

Reply:
Thanks